customer-commons/src/main/java/no/unit/nva/customer/model/CustomerList.java

```diff
@@ -30,7 +30,11 @@ public CustomerList(List<CustomerDto> customers) {
      * @return customerList
      */
     public static CustomerList of(List<CustomerDto> customers) {
-        return new CustomerList(customers);
+        List<CustomerDto> customersWithoutContext = customers
```

Comment:
Moved functionality of CustomerMapper here. This creates a "null" context for each entry in the list, but this is a bug that will be fixed in another PR. In any case, the fix is now local to this class.